e2e/run-e2e-suite.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+KIND_LOG_LEVEL="info"
+
+if ! [ -z "$DEBUG" ]; then
+	set -x
+  KIND_LOG_LEVEL="debug"
+fi
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+RED='\e[35m'
+NC='\e[0m'
+BGREEN='\e[32m'
+
+K8S_VERSION=${K8S_VERSION:-v1.15.3}
+KIND_CLUSTER_NAME="external-secrets-dev"
+REGISTRY=external-secrets
+
+kind --version || $(echo -e "${RED}Please install kind before running e2e tests${NC}";exit 1)
+echo -e "${BGREEN}[dev-env] creating Kubernetes cluster with kind${NC}"
+
+kind create cluster \
+  --loglevel=${KIND_LOG_LEVEL} \
+  --name ${KIND_CLUSTER_NAME} \
+  --config ${DIR}/kind.yaml \
+  --image "kindest/node:${K8S_VERSION}"
+
+export KUBECONFIG="$(kind get kubeconfig-path --name="${KIND_CLUSTER_NAME}")"
+
+echo -e "${BGREEN}building external-secrets images${NC}"
+docker build -t external-secrets:test -f $DIR/../Dockerfile $DIR/../
+docker build -t external-secrets-e2e:test -f $DIR/Dockerfile $DIR/../
+kind load docker-image --name="${KIND_CLUSTER_NAME}" external-secrets-e2e:test
+kind load docker-image --name="${KIND_CLUSTER_NAME}" external-secrets:test
+
+function cleanup {
+  set +e
+  kubectl delete pod e2e 2>/dev/null
+  kubectl delete crd/externalsecrets.kubernetes-client.io 2>/dev/null
+  kubectl delete -f ${DIR}/localstack.deployment.yaml 2>/dev/null
+  kind delete cluster \
+    --loglevel=${KIND_LOG_LEVEL} \
+    --name ${KIND_CLUSTER_NAME}
+
+}
+trap cleanup EXIT
+
+kubectl apply -f ${DIR}/localstack.deployment.yaml
+
+helm template $DIR/../charts/kubernetes-external-secrets \
```

Comment:
worth passing setting --name?  e.g., `helm template --name e2e`.

then you can avoid relying on `release-name` as the default prefix for the deployment name